great-idea-website/index.html

```diff
@@ -14,50 +15,84 @@
   <![endif]-->
 </head>
 
-  <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+<body>
+
+  <header class="container">
+    <nav>
+      <a href="">Services</a>
+      <a href="">Product</a>
+      <a href="">Vision</a>
+      <a href="">Features</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+    </nav>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </header>
+
+
+  <section>
+    <div class="left">
+      <h1>
+        Innovation
+        On
+        Demand
+      </h1>
+
+      <button>
+        Get Started
+      </button>
+    </div>
+    <div class="right">
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </section>
+
+  <section class="top-content">
+    <h5>Features</h5>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+
+    <h5>About</h5>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </section>
 
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
 
-  Get Started
+  <section class="bottom-content">
+    <h5>Services</h5>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
 
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    <h5>Product</h5>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendiss eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
 
+    <h5>Vision</h5>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendiss eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </section>
 
```

Comment:
```suggestion
  <section class="bottom-content">
    <h5>Services</h5>
    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
   <img src="img/header-img.png" alt="Image of a code snippet.">
  Features
  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
    <h5>Product</h5>
    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
      Suspendiss eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
     <h5>Vision</h5>
    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
      Suspendiss eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
  </section>
``` like aboce they need seperate divs